cmd/backup/script.go

```diff
@@ -89,6 +89,18 @@ func newScript() (*script, error) {
 		return nil
 	})
 
+	envconfig.Lookup = func(key string) (string, bool) {
+		location, ok := os.LookupEnv(key + "_FILE")
+		if ok {
+			contents, err := os.ReadFile(location)
+			if err != nil {
```

Comment:
Since (I think) we just agreed in the last comment on preferring the value over the _FILE variant, it would mean that the user specifically wants to use a _FILE and that failed -> undefined/undesired behavior for the user -> panic on any error.